common/scala/src/main/resources/application.conf

```diff
@@ -155,12 +155,44 @@ whisk {
 
     # CosmosDB related configuration
     # For example:
-    # cosmosdb {
-    #     endpoint   =               # Endpoint URL like https://<account>.documents.azure.com:443/
-    #     key        =               # Access key
-    #     db         =               # Database name
-    #     throughput = 1000          # Throughput configure for each collection within this db
-    #}
+    cosmosdb {
+    #   endpoint          =               # Endpoint URL like https://<account>.documents.azure.com:443/
+    #   key               =               # Access key
+    #   db                =               # Database name
+        throughput        = 1000          # Throughput configured for each collection within this db
```

Comment:
Does this correspond to RU config at azure? (do these need to be in sync?)

Reply:
Its used only when a fresh collection is being created. If collection already exists then its throughput is not modified